content/manuals/billing/cycle.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Change your billing cycle
+weight: 50
+description: Learn to change your billing cycle for your Docker subscription
+keywords: billing, cycle, payments, subscription
+aliases:
+- /billing/cycle/
+---
+
+You can pay for a subscription plan on a monthly or yearly billing cycle. You select your preferred billing cycle when you buy your subscription.
+
+> [!NOTE]
+>
+> Business plan is available only on yearly billing cycle.
+
+If you have a monthly billing cycle, you can choose to switch to an annual billing cycle.
+
+> [!NOTE]
+>
+> You can't switch from an annual billing cycle to a monthly cycle.
+
+When you change the billing cycle's duration:
+
+- The next billing date reflects the new cycle. To find your next billing date, see [View renewal date](history.md#view-renewal-date).
+- The subscription's start date resets. For example, if the start date of the monthly subscription is March 1st and the end date is April 1st, then after switching the billing duration to March 15th, 2023 the new start date is March 15th, 2023, and the new end date is March 15th, 2024.
```

Comment:
**nitpick**: Since these are new docs and it is already 2024, should these dates be revised to 2024 and 2025?